lte/gateway/c/core/oai/tasks/nas5g/include/ies/M5GGprsTimer.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * */
+#pragma once
+#include <sstream>
+#include <cstdint>
+namespace magma5g {
+class GPRSTimerMsg {
+ public:
+#define GPRS_TIMER_MINIMUM_LENGTH 2
+#define GPRS_TIMER_MAXIMUM_LENGTH 2
+
+  uint8_t iei;
+  uint8_t timervalue;
+  GPRSTimerMsg();
+  ~GPRSTimerMsg();
+
+  int EncodeGPRSTimerMsg(GPRSTimerMsg* gprstimer, uint8_t iei, uint8_t* buffer,
+                         uint32_t len);
+
+  int DecodeGPRSTimerMsg(GPRSTimerMsg* gprstimer, uint8_t iei, uint8_t* buffer,
+                         uint32_t len);
```

Comment:
Do not see these functions are used in either Functional or UT code. Can you check this once, if needed for any future use can enclose this in #if 0 for now

Reply:
This is added for the future purpose